src/main/scala/org/broadinstitute/dsde/agora/server/webservice/util/ServiceMessages.scala

```diff
@@ -19,6 +19,11 @@ object ServiceMessages {
                                         username: String,
                                         accessObject: AccessControl)
 
+  case class InsertBatchNamespacePermission(requestContext: RequestContext,
```

Comment:
Maybe just call this BatchNamespacePermission, since it can also be used to delete.